documentation/source/user_section/installation.rst

```diff
@@ -96,8 +96,120 @@ In the context of SCT, it can be used:
+
+CentOS7-based installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+   # Start from the Terminal
+   docker pull centos:centos7
+   # Launch interactive mode (command-line inside container)
+   docker run -it centos:centos7
+   # Now, inside Docker container, install dependencies
+   yum install -y which gcc git curl
+   # Install SCT
+   git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+   cd sct
+   yes | ./install_sct
+   export PATH="/sct/bin:${PATH}"
+   # Test SCT
+   sct_testing
+   # save the state of the container. Open a new Terminal and run:
+   docker ps -a  # list all containers
+   docker commit <CONTAINER_ID> <YOUR_NAME>/centos:centos7
+
+
+Using GUI Scripts (Optional)
+============================
+
+In order to run scripts with GUI you need to allow X11 redirection.
+First, save your Docker image:
+
+1. Open another Terminal
+2. List current docker images
+
+.. code:: bash
+
+   docker ps -a
+
+3. Save container as new image
+
+.. code:: bash
+
+   docker commit <CONTAINER_ID> <YOUR_NAME>/<DISTROS>:<VERSION>
+
+For macOS and Linux users
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create an X11 server for handling display:
+
+1. Install XQuartz X11 server.
+2. Check ‘Allow connections from network clientsoption inXQuartz\`
+   settings.
+3. Quit and restart XQuartz.
+4. In XQuartz window xhost + 127.0.0.1
+5. In your other Terminal window, run:
+
+   -  On macOS:
+      ``docker run -e DISPLAY=host.docker.internal:0 -it <CONTAINER_ID>``
+   -  On Linux:
+      ``docker run -ti --rm -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix <CONTAINER_ID>``
+
+For Windows users
+~~~~~~~~~~~~~~~~~
+
+1. Install Xming
+2. Connect to it using Xming/SSH:
+
+  - If you are using Docker Desktop, please download and run (double click) the following script: :download:`sct-win.xlaunch<../../../contrib/docker/sct-win.xlaunch>`.
+  - If you are using Docker Toolbox, please download and run the following script instead: :download:`sct-win_docker_toolbox.xlaunch<../../../contrib/docker/sct-win_docker_toolbox.xlaunch>`
+  - If this is the first time you have done this procedure, the system will ask you if you want to add the remote PC (the docker container) as trust pc, type yes. Then type the password to enter the docker container (by default sct).
+
+**Troubleshooting:**
+
+| If there are no new open windows:
+| - Please download and run the following file:
+  :download:`Erase_fingerprint_docker.sh<../../../contrib/docker/Erase_fingerprint_docker.sh>`
+| - Try again
+| - if it is still not working :
+| - Open the file manager and go to C:/Users/Your_username - In the
+  searchbar type ‘.ssh’ - Open the found ‘.ssh’ folder. - Open the
+  ‘known_hosts’ file with a text editor - Remove line starting with
+  ``192.168.99.100`` or ``localhost`` - try again
+
+The graphic terminal emulator LXterminal should appear (if not check the
+task bar at the bottom of the screen), which allows copying and pasting
+commands, which makes it easi
```

Comment:
hum, i feel the copy/paste was cropped? This chunk seems to be missing:

![image](https://user-images.githubusercontent.com/2482071/97377304-1b78e180-1896-11eb-92e5-b8cdb4c7f81f.png)

Reply:
You're absolutely right. I had used a `.md` to `.rst` converter, and I had trusted it blindly rather than checking it carefully. Thank you for catching this.

(I think this is the cause of the issues above, as well.)

I'll go through the conversion process more carefully this time around. :)